core/bootstrap/src/main/java/org/jboss/modules/maven/MavenArtifactUtil.java

```diff
@@ -18,6 +18,15 @@
 
```

Comment:
Maybe an alternative approach is to create a parent class, or interface, with those methods that the Maven and Gradle helpers then use.

That way it's clear there's common functionality being reused vs Gradle using something from Maven, which isn't the case but could look that way from the calls.

If you could undo the import changes that would be great also.

Thanks